### DISCUSSION TITLE
HW 2 part 3

### DISCUSSION THREAD

**Discussion by ElisDiaz46** [cooperrc/computational-mechanics#179] [Q&A]
How come my data is on the x-axis in the graph? 

Here is my code 

![image](https://user-images.githubusercontent.com/116235922/212369588-7db085f5-b69b-4ce2-a33c-7b20108b2d8d.png)
![image](https://user-images.githubusercontent.com/116235922/212369727-cbcfefd6-055b-40aa-84b0-ad864428f537.png)

**Comment by Awasthi-Adi** [comment]
I would suggest going back and renaming variables with names of what is actually in them. For instance; US_p (for array of USA teraWatt-hours), year (for the array of years), etc. It seems that you are mixing variables up and that is effecting your graph. Cool idea to create a function to determine your coefficients for you. The underlying math looks good.
